src/styles/stars.scss

```
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+// .stars--1 > .stars__star:nth-child(-n + 1),
+// .stars--2 > .stars__star:nth-child(-n + 2),
+// .stars--3 > .stars__star:nth-child(-n + 3),
+// .stars--4 > .stars__star:nth-child(-n + 4),
+// .stars--5 > .stars__star:nth-child(-n + 5) {
+//   background-image: url(../images/star-active.svg);
+//   background-repeat: no-repeat;
+// }
+
+@for $starCount from 1 through 5 {
+  .stars--#{$starCount} .stars:nth-child(-n + #{$starCount}) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

This may help you to display active stars
```suggestion
@for $starCount from 1 through 5 {
  .stars--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
    background-image: url(../images/star-active.svg);
  }
}
```